.chocolatey/tools/ChocolateyInstall.ps1

```diff
@@ -51,6 +51,18 @@ $packageArgs = @{
 
 Install-ChocolateyZipPackage @packageArgs
 
+$TempSettingsPath = Join-Path $env:TEMP 'PresenceLightSettings.json'
+
+if (Test-Path -Path $TempSettingsPath -PathType Leaf)
+{
+    $SettingsPath Join-Path $InstallDir 'appsettings.json'
+    Remove-Item -Path $SettingsPath -Recurse
+    
+    Get-ChildItem -path $TempSettingsPath | 
+    Copy-Item -Destination $InstallDir -PassThru | Rename-Item -NewName  'appsettings.json'
```

Comment:
Same comment here, it was the `settings.json` file that stored the custom settings from the UI.  At least in the test that I did.